src/microbe_stage/MicrobeCamera.cs

```diff
@@ -269,9 +237,7 @@ public void UpdateCameraPosition(double delta, Vector3? followedObject)
 
         if (backgroundPlane != null)
```

Comment:
`backgroundPlane` cannot be null so this if condition should be removed (I'll be surprised a bit if the CI didn't catch this).